docs/authorization.md

```diff
@@ -101,3 +101,106 @@ to a remote authorization service.  Authorization modules can implement
 their own caching to reduce the cost of repeated authorization calls with the
 same or similar arguments.  Developers should then consider the interaction between
 caching and revocation of permissions.
+
+
+## External APIs
+
+### /api/{version}/ns/{namespace}/subjectAccessReview
+This API answers the question: can a user or group (use authenticated user if none is specified) perform a given action?
+
+SubjectAccessReview is runtime.Object with associated RESTStorage that only accepts creates.  The caller POSTs a SubjectAccessReview to this URL with the `spec` values filled in.  He gets a SubjectAccessReview back, with the `status` values completed.  Here is an example of a call and its corresponding return.
```

Comment:
I don't see the benefit of following the runtime.Object pattern with spec and status and RESTStorage and all that.  This is more of a query.  GET seems more appropriate than POST since it doesn't create a resource that you can subsequently get.